### PR TITLE
[130] Add photo to freaks

### PR DESCRIPTION
**Pull request by salcudeann**
## Description
Create a Photo Model and a table in database for photos.
For each freak will be saved in database an url to his profile picture, which will pe uploaded on [Gravatar](https://ro.gravatar.com/site/implement)

## Related

[[130] Add Photo to a Freak](https://trello.com/c/0yKfo1ER/130-add-photo-to-a-freak)

## Steps to Test or Reproduce
You can run a test from here `spec/controllers/graphql/mutations/photo_create_spec.rb`.
Also, you can run the test with Insomnia or by executing a GraphQl mutation from localhost:

1. Start rails server by typing `rails s `command in terminal
2. Access [localhost/graphiql](http://127.0.0.1:3000/graphiql) or use Insomnia to execute a GraphQl mutation
3. By executing this mutation you can see that everything works fine
```GraphQl
mutation {
    photoCreate(uri: "https://www.gravatar.com/avatar/00000000000000000000000000000000") {
        uri
    }
}
`

### CODE REVIEW

**Review by alex-damian-negru — DISMISSED** [review]
This seems to have commits from 3 PRs. Please remove the unneeded PRs.

File: app/graphql/types/mutation_type.rb [hunk 1]
```diff
@@ -1,7 +1,8 @@
 # frozen_string_literal: true
 
 module Types
-  class MutationType < Types::BaseObject
-    field :create_freak, FreakType, null: false, resolver: Resolvers::FreakCreateResolver
+  class MutationType < BaseObject
+    field :freak_create, FreakType, null: false, resolver: Resolvers::FreakCreateResolver
+    field :photo_create, Models::Photo, null: false, resolver: Resolvers::PhotoCreateResolver
```
Comment (pirvudoru) [inline app/graphql/types/mutation_type.rb:6]: Interesting that you added this. We can keep it, but we didn't need it yet.

File: db/migrate/20210802074146_create_freaks.rb [hunk 2]
```diff
@@ -5,7 +5,7 @@ def change
     create_table :freaks do |t|
       t.string :name
       t.string :description
-      t.date :photo
+      t.string :photo
```
Comment (pirvudoru) [inline db/migrate/20210802074146_create_freaks.rb:8]: We need to remove this column from the freaks table, a photo is modeled as a separate table.

File: spec/models/photo_spec.rb [hunk 3]
```diff
@@ -0,0 +1,7 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe Photo, type: :model do
+  pending "add some examples to (or delete) #{__FILE__}"
```
Comment (pirvudoru) [inline spec/models/photo_spec.rb:6]: Do as it says.

**Review by pirvudoru — CHANGES_REQUESTED** [review]

File: app/graphql/types/mutation_type.rb (hunk 1 above)
Comment (salcudeann) [reply app/graphql/types/mutation_type.rb:6]: We need this field for photoCreate mutation, which is used by `spec/controllers/graphql/mutations/photo_create_spec.rb` test.

File: db/migrate/20210802074146_create_freaks.rb (hunk 2 above)
Comment (salcudeann) [reply db/migrate/20210802074146_create_freaks.rb:8]: Done!

File: spec/models/photo_spec.rb (hunk 3 above)
Comment (salcudeann) [reply spec/models/photo_spec.rb:6]: I think this file should be removed, I've already created a test to create a photo entry in database.

**Pull request closed without merging**